tests/utils/Statistic/total-minus-loads-stats.test.js

```diff
@@ -0,0 +1,12 @@
+import { describe, expect, test } from 'vitest';
+import data from "./interpolate-data-test.json" assert { type: 'json' };
+import totalMinusLoadsStats from '$lib/utils/Statistic/total-minus-loads-stats';
+
+
+const loadFts = ['exports', 'battery_charging', 'pumps'];
+
+describe("Test ", () => {
```

Comment:
please describe the test.

Reply:
Sorry about that, this one is still in-progress